src/types.d.ts

```diff
@@ -123,6 +124,7 @@ export interface SchemaElement {
   precision?: number
   field_id?: number
   logical_type?: LogicalType
+  geospatial?: boolean // if the column contains geospatial data
```

Comment:
this does not exist on the parquet SchemaElement type?
https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L505

Reply:
Exactly. But I don't know how we should work around that. To decode, we only use the schema. An alternative I was thinking of was to set the logical type to GEOMETRY. But it would be fake.
What would you propose?

Reply:
Oh, maybe with the custom parsers!

Reply:
Also: to avoid the kind of "utf8=true" issue, maybe we want automatic conversion to GeoJSON in case of GeoParquet to be an option, not the default behavior?